frontend/apps/picasso/apollo/queries/stakingPositions.ts

```diff
@@ -17,18 +17,20 @@ export interface StakingPositions {
 }
 
 export const GET_STAKING_POSITIONS = gql`
-    query stakingPositionsByOwner($accountId: String) {
-        stakingPositions(orderBy: fnftCollectionId_ASC, where: {owner_eq: $accountId}) {
-            amount
-            assetId
-            endTimestamp
-            fnftCollectionId
-            fnftInstanceId
-            id
-            owner
-            source
-            startTimestamp
-        }
+  query stakingPositionsByOwner($accountId: String) {
```

Comment:
This will likely change, kept it as reference to know what fields are required to change once subsquid changes are done.